src/pool/srv_pool.c

```diff
@@ -2432,6 +2432,15 @@ ds_pool_svc_list_cont(uuid_t uuid, d_rank_list_t *ranks,
 	return rc;
 }
 
+static bool
+is_rpc_from_client(crt_rpc_t *rpc)
+{
+	d_rank_t srcrank;
+
+	crt_req_src_rank_get(rpc, &srcrank);
+	return (srcrank == -1);
```

Comment:
With a recent CaRT update and with CaRT PR https://github.com/daos-stack/cart/pull/309 the CRT_NO_RANK constant has now been exposed in cart/types.h so we can change the -1 test now to use this constant.

Reply:
Excellent. Fixed.

Reply:
Excellent. Fixed.